winui/Kanban/column.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Columns in WinUI Kanban control | Syncfusion
+description: Learn here all about configuring and customizing columns in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Column in WinUI Kanban Control
+
+The WinUI Kanban control organizes its layout into columns, representing distinct stages of a workflow or process. Common stages include To-Do, In-Progress, Testing, Validation, and Completed.
+
+## Customizing column width
+
+Columns are smartly sized by default to ensure optimal arrangement and readability of card elements. However, you can customize column widths to suit your specific requirements by using `MinimumColumnWidth` and `MinimumColumnWidth` properties of `SfKanban`.
```

Comment:
By default, the columns are sized smartly to arrange the default elements of the cards with better readability. You can also define the minimum and maximum widths for the columns in SfKanban using the `MinimumColumnWidth` and `MaximumColumnWidth` properties, respectively.